packages/intrepid2/src/Shared/Intrepid2_RealSpaceTools.hpp

```diff
@@ -93,7 +93,7 @@ namespace Intrepid2 {
       KOKKOS_INLINE_FUNCTION
       static inVecValueType
       vectorNorm( const Kokkos::DynRankView<inVecValueType,inVecProperties...> inVec, 
-                  const ENorm normType );
+                  const ENorm normType, bool bFence = true );
```

Comment:
This is Kokkos inline function which can be used inside of a functor. We cannot put a fence here.